api/test/unit/utils/test_image_proxy.py

```diff
@@ -10,18 +10,14 @@
 import pytest
 import requests
 
-from api.utils.photon import (
-    HEADERS,
-    UpstreamThumbnailException,
-    _get_file_extension_from_url,
-    check_image_type,
-)
-from api.utils.photon import get as photon_get
+from api.utils.image_proxy import HEADERS, UpstreamThumbnailException, extension
+from api.utils.image_proxy import get as photon_get
 from api.utils.tallies import get_monthly_timestamp
 
 
 PHOTON_URL_FOR_TEST_IMAGE = f"{settings.PHOTON_ENDPOINT}subdomain.example.com/path_part1/part2/image_dot_jpg.jpg"
 TEST_IMAGE_URL = PHOTON_URL_FOR_TEST_IMAGE.replace(settings.PHOTON_ENDPOINT, "http://")
+TEST_IMAGE_IDENTIFIER = "123"
```

Comment:
TODO: rename this to `TEST_MEDIA_IDENTIFIER` because it can be an audio thumbnail, too.